<launch>

  <param name="path_to_objects_models" type="str" value="$(find shopping_list)/data/objects_models.yml" />
  <param name="akaze_threshold" value="3e-4" type="double" />
  <param name="ransac_threshold" value="5.0" type="double" />
  <param name="nn_match_ratio" value="0.65" type="double" />
  <param name="bb_min_inliers" value="10" type="int" />

  <node name="shopping_list_server" pkg="shopping_list" type="shopping_list_node" output="screen" />

</launch>
